proto/echo/echo.proto

```
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package echo; 
```

Buf (1.70.0)

[error] Files with package "echo" must be within a directory "echo" relative to root but were in directory "proto/echo".

(PACKAGE_DIRECTORY_MATCH)